e2e/image/pull_test.go

```diff
@@ -28,6 +30,13 @@ func TestPullWithContentTrust(t *testing.T) {
 	golden.Assert(t, result.Stdout(), "pull-with-content-trust.golden")
 }
 
+func TestPullQuiet(t *testing.T) {
+	result := icmd.RunCommand("docker", "pull", "--quiet", fixtures.AlpineImage)
+	result.Assert(t, icmd.Success)
+	assert.Check(t, is.Equal(result.Stdout(), ""))
```

Comment:
Hm, I don't think it's correct that _nothing_ is output; it should print the image name and/or sha after pull completed. Perhaps we can combine, i.e.:

```
busybox:latest@sha256:58ac43b2cc92c687a32c8be6278e50a063579655fe3090125dcb2af0ff9e1a64
```

Reply:
Wouldn't that defeat the purpose of a `quiet` option?